examples/NetCore2/ConsoleExample/Program.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using NLog.Extensions.Logging;
+
+namespace ConsoleExample
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var servicesProvider = BuildDi();
+            var runner = servicesProvider.GetRequiredService<Runner>();
+
+            runner.DoAction("Action1");
+
+            Console.WriteLine("Press ANY key to exit");
+            Console.ReadLine();
+        }
+
+
+        private static IServiceProvider BuildDi()
+        {
+            var services = new ServiceCollection();
+
+            services.AddTransient<Runner>();
+            // services.AddSingleton<ILoggerFactory, NLogLoggerFactory>();
+            services.AddSingleton<ILoggerFactory, LoggerFactory>();
```

Comment:
@snakefoot any idea why the structured logging won't work if I use

     services.AddSingleton<ILoggerFactory, NLogLoggerFactory>();

instead of
  
    services.AddSingleton<ILoggerFactory, LoggerFactory>();

(also in unit tests)

Reply:
Don't know much of about this, so no